tests/test_admin.py

```diff
@@ -44,7 +44,7 @@ def test_get_queryset(self):
         request.user = self.admin
 
         qs = sub_org_inline.get_queryset(request)
-        self.assertQuerysetEqual(qs, [repr(self.editable_org)])
+        self.assertQuerysetEqual(qs, [self.editable_org])
```

Comment:
There's deprecation warnings like these:
``` /home/runner/work/django-orghierarchy/django-orghierarchy/tests/test_admin.py:47: RemovedInDjango51Warning: assertQuerysetEqual() is deprecated in favor of assertQuerySetEqual().```

Not exactly required since we're only concerned with Django compatibility up to 4.2, but might as well do a quick replace all for `assertQuerysetEqual` -> `assertQuerySetEqual`

Reply:
I actually tried this and it fails on older Django versions. The new `assertQuerySetEqual` method (and deprecation) was added in 4.2 https://github.com/django/django/commit/f0c06f8ab7904e1fd082f2de57337f6c7e05f177

Reply:
Ah, I see. Fair enough!